README.md

```diff
@@ -79,379 +77,373 @@ All URIs are relative to *https://api.synctera.com/v0*
-*WatchlistApi* | [**ListWatchlistAlerts**](synctera/docs/WatchlistApi.md#listwatchlistalerts) | **Get** /customers/{customer_id}/watchlists/alerts | List watchlist monitoring alerts for a customer
-*WatchlistApi* | [**ListWatchlistSubscriptions**](synctera/docs/WatchlistApi.md#listwatchlistsubscriptions) | **Get** /customers/{customer_id}/watchlists/subscriptions | List watchlist monitoring subscriptions for a customer
-*WatchlistApi* | [**SuppressWatchlistEntityAlert**](synctera/docs/WatchlistApi.md#suppresswatchlistentityalert) | **Post** /customers/{customer_id}/watchlists/suppressions | Suppress entity alert
-*WatchlistApi* | [**UpdateWatchlistAlert**](synctera/docs/WatchlistApi.md#updatewatchlistalert) | **Put** /customers/{customer_id}/watchlists/alerts/{alert_id} | Update watchlist alert
-*WatchlistApi* | [**UpdateWatchlistSubscription**](synctera/docs/WatchlistApi.md#updatewatchlistsubscription) | **Put** /customers/{customer_id}/watchlists/subscriptions/{subscription_id} | Update watchlist monitoring subscription
-*WatchlistApi* | [**WatchlistSubscribe**](synctera/docs/WatchlistApi.md#watchlistsubscribe) | **Post** /customers/{customer_id}/watchlists/subscriptions | Subscribe a customer to watchlist monitoring
-*WebhooksApi* | [**CreateSecret**](synctera/docs/WebhooksApi.md#createsecret) | **Post** /webhooks/secret | Create a secret
-*WebhooksApi* | [**CreateWebhook1**](synctera/docs/WebhooksApi.md#createwebhook1) | **Post** /webhooks | Create a webhook
-*WebhooksApi* | [**DeleteWebhook**](synctera/docs/WebhooksApi.md#deletewebhook) | **Delete** /webhooks/{webhook_id} | Delete a webhook
-*WebhooksApi* | [**GetEvent**](synctera/docs/WebhooksApi.md#getevent) | **Get** /webhooks/{webhook_id}/events/{event_id} | Get webhook event
-*WebhooksApi* | [**GetWebhook1**](synctera/docs/WebhooksApi.md#getwebhook1) | **Get** /webhooks/{webhook_id} | Get a webhook
-*WebhooksApi* | [**ListEvents**](synctera/docs/WebhooksApi.md#listevents) | **Get** /webhooks/{webhook_id}/events | List webhook events
-*WebhooksApi* | [**ListWebhooks1**](synctera/docs/WebhooksApi.md#listwebhooks1) | **Get** /webhooks | List webhooks
-*WebhooksApi* | [**ReplaceSecret**](synctera/docs/WebhooksApi.md#replacesecret) | **Put** /webhooks/secret | Replace an existing secret
-*WebhooksApi* | [**ResendEvent**](synctera/docs/WebhooksApi.md#resendevent) | **Post** /webhooks/{webhook_id}/events/{event_id}/resend | Resend an event
-*WebhooksApi* | [**RevokeSecret**](synctera/docs/WebhooksApi.md#revokesecret) | **Delete** /webhooks/secret | Revoke the secret
-*WebhooksApi* | [**TriggerEvent**](synctera/docs/WebhooksApi.md#triggerevent) | **Post** /webhooks/trigger | Trigger an event
-*WebhooksApi* | [**UpdateWebhook**](synctera/docs/WebhooksApi.md#updatewebhook) | **Put** /webhooks/{webhook_id} | Update a webhook
+*AccountsApi* | [**CreateAccount**](docs/AccountsApi.md#createaccount) | **Post** /accounts | Create an account
```

Comment:
These paths won't route from the README

Reply:
do they not? Worked fine for me :thinking:

Reply:
Sorry, I see now you eliminated the synctera/ path which was in the former archive